core/common/src/main/java/alluxio/exception/JobDoesNotExistRuntimeException.java

```diff
@@ -11,19 +11,21 @@
 
 package alluxio.exception;
 
+import io.grpc.Status;
+
 /**
- * The exception thrown when a block does not exist in Alluxio.
+ * The exception thrown when a job definition does not exist in Alluxio.
  */
-public class BlockDoesNotExistException extends AlluxioException {
-  private static final long serialVersionUID = -1313208091357063634L;
+public class JobDoesNotExistRuntimeException extends AlluxioRuntimeException {
```

Comment:
What is this change about?

Reply:
I just create a new JobDoesNotExistRuntimeException for later usage and delete the unused BlockDoesNotExistException.